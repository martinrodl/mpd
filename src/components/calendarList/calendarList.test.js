import mockData from './mockEvents';
import CalendarList from './calendarList';
import { render, screen } from '@testing-library/react';
import React from 'react';

test('Consistency of mockData', () => {
  expect(mockData).toMatchSnapshot();
});
