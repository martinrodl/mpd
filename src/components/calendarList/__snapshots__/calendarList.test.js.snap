// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Consistency of mockData 1`] = `
Array [
  Object {
    "end": "2020-08-07 01:30:00",
    "id": 0,
    "start": "2020-08-07 00:30:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-07 02:20:00",
    "id": 1,
    "start": "2020-08-07 01:30:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-07 04:40:00",
    "id": 2,
    "start": "2020-08-07 04:10:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-07 01:45:00",
    "id": 3,
    "start": "2020-08-07 01:05:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-07 16:30:00",
    "id": 4,
    "start": "2020-08-07 14:30:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-08 02:20:00",
    "id": 5,
    "start": "2020-08-08 01:20:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-08 04:40:00",
    "id": 6,
    "start": "2020-08-08 04:10:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-08 01:45:00",
    "id": 7,
    "start": "2020-08-08 00:45:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-08 12:30:00",
    "id": 8,
    "start": "2020-08-08 11:30:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-09 02:00:00",
    "id": 9,
    "start": "2020-08-09 01:30:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-09 03:40:00",
    "id": 10,
    "start": "2020-08-09 03:10:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-09 01:45:00",
    "id": 11,
    "start": "2020-08-09 00:10:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
  Object {
    "end": "2020-08-04 01:45:00",
    "id": 12,
    "start": "2020-08-04 00:10:00",
    "summary": "3412 Piedmont Rd NE, GA 3032",
    "title": "Dr. Mariana Joseph",
  },
]
`;
